ea_restaurant_data_loader/lib/service/oauth2_service.ex

```diff
@@ -0,0 +1,80 @@
+defmodule EaRestaurantDataLoader.Oauth2Service do
+  alias EaRestaurantDataLoader.Repo
+  alias EaRestaurantDataLoader.AppClient
+  alias EaRestaurantDataLoader.AppClientScope
+  alias EaRestaurantDataLoader.AppAccessToken
+  alias EaRestaurantDataLoader.AppRefreshToken
+  alias EaRestaurantDataLoader.Oauth2Util
+  import Ecto.Query
+
+  def login_client(client_id, client_secret) do
+    secret_key = Application.get_env(:ea_restaurant_data_loader, :secret_key)
+
+    validate_client_credentials(client_id, client_secret, "ACTIVE")
```

Comment:
Can put all of these contants into a file to be reused?

Reply:
Done.